factory-reset/eos-factory-reset

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python3
+import argparse
+import logging
+import subprocess
+import sys
+
+from systemd import journal
+
+# Options list
+# Each entry has the form (short name, long name, description)
+OPTS = [
+    ('D', 'disable', 'fully remove a scheduled factory reset'),
+]
+
+# Stages dictionary
+# Each entry has the form {name: description}
+# When creating a new stage make sure to check both STAGES and OPTS so names
+# don't clash, as they both create optional command line arguments.
+STAGES = {
+    'users': 'remove all user accounts (and home directories)',
+}
+
+
+def yes_or_no(question):
+    answer = ''
+    while answer not in ['y', 'n']:
+        answer = input(question + ' ').lower()
+    return answer == 'y'
+
+
+def toggle_stage(logger, stage, action):
+    unit = 'eos-factory-reset-' + stage + '.service'
+    cmd = ['systemctl', '--quiet', action]
+    cmd.append(unit)
+    sp = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    status, msg = sp.returncode, sp.stdout.decode().rstrip('\n')
+    if status != 0:
+        logger.error('Failed to %s stage \'%s\': %s', action, stage, msg)
```

Comment:
```suggestion
        logger.error("Failed to %s stage '%s': %s", action, stage, msg)
```

Reply:
I applied this manually and changed all string quotation marks from single to double, to keep it consistent.